app/session/routes.py

```diff
@@ -24,8 +23,8 @@ def post_session():
     if current_user:
         user_uuid = current_user.user_uuid
 
-    store_session(session_uuid, session_created_timestamp, user_uuid)
-    process_ip_address(request, session_uuid)
+    ip_address = get_ip_address(request)
+    store_session(session_uuid, session_created_timestamp, user_uuid, ip_address)

```

Comment:
instead of 
- create
- get IP 
- save again
we can use just: 
- get IP
- create